<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Formulario</class>
 <widget class="QWidget" name="Formulario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>321</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ataque</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="titulo">
      <property name="text">
       <string>Ingrese los datos de su Luchador</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Velocidad</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="NombreLuchador"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Defensa</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QPushButton" name="Agregar">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Agregar</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="BotonSalir">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="VelocidadLuchador"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="AtaqueLuchador"/>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="DefensaLuchador"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
